indico/web/templates/forms/permissions_widget.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html %}
+    <h3>Permissions</h3>
+    <div id="permissions-widget-{{ field.id }}" class="permissions-widget">
+        {% set event = field.event %}
+        <input type="hidden" value='{{ field._value()|tojson|forceescape }}' name="{{ field.name }}">
+        <ul class="permissions-widget-list"></ul>
+        <div class="toolbars">
+            <div class="toolbar">
+                <div class="group">
+                    <a href="#" class="i-button label text-color outline add-entry-decoration">
+                        {%- trans %}Add{% endtrans -%}
+                    </a>
+                    <a class="i-button js-add-user-group">{% trans %}User / Group{% endtrans %}</a>
+                    <a class="i-button arrow js-dropdown" data-toggle="dropdown">{% trans %}Role{% endtrans %}</a>
+                    <ul class="dropdown entry-role-dropdown" data-items='{{ field.roles|tojson|forceescape }}'>
+                        <li class="separator default"></li>
+                        <li class="entry-item new-role-btn default">
+                            <a class="js-new-role"
+                               data-href="{{ url_for('event_roles.add_role', event) }}"
+                               data-title="{% trans %}Add a new role{% endtrans %}"
+                               data-ajax-dialog>
+                                <span class="icon-medal dropdown-icon"></span>
+                                <span>{% trans %}New role{% endtrans %}</span>
+                            </a>
+                        </li>
+                    </ul>
+                    {% if field.ip_networks %}
+                        <a class="i-button arrow js-dropdown" data-toggle="dropdown">
+                            {%- trans %}IP Network{% endtrans -%}
+                        </a>
+                        <ul class="dropdown entry-ip-network-dropdown"
+                            data-items='{{ field.ip_networks|tojson|forceescape }}'></ul>
+                    {% endif %}
+                </div>
+            </div>
+            <div class="toolbar">
+                <a href="{{ url_for('event_roles.manage', event) }}"
+                   class="i-button icon-medal text-color borderless">
+                    {%- trans %}Manage roles{% endtrans -%}
```

Comment:
IIRC. we wanted to remove this button. It's not very useful now that you can add roles from the dropdown, and clicking it results in you losing your changes in the form (which is not obvious beforehand).